packages/integration-tests/suites/public-api/addBreadcrumb/empty_obj/test.ts

```diff
@@ -1,7 +1,6 @@
 import { expect } from '@playwright/test';
-
-import { sentryTest } from '../../../../utils/fixtures';
-import { getSentryRequest } from '../../../../utils/helpers';
+import { sentryTest } from '@utils/fixtures';
+import { getSentryRequest } from '@utils/helpers';
```

Comment:
tbh I don't see the value in the relative import here, it just makes things more confusing. Any thoughts?

Reply:
Yes, actually I thought so. But as we're introducing this alias for subjects, it seemed less confusing to have the same behaviour on the assertions. I'm totally fine with scraping aliases for assertions. It requires too much custom configuration anyways.